bin/fin

```diff
@@ -1240,11 +1240,16 @@ show_help_list_commands_in_help ()
 	else
 		addons_path="$HOME/$DOCKSAL_COMMANDS_PATH"
 	fi
-
-	local addons_list=$(find "$addons_path" -type f -perm +100 2>/dev/null)
+	
+	local executable="-executable"
+	if [[ is_mac ]]; then
+		executable="-perm +100"
+	fi
+	
+	local addons_list=$(find "$addons_path" -type f ${executable} 2>/dev/null)
 	if [ ! -z "$addons_list" ]; then
```

Comment:
We are using bash style tests instead of sh, so `if [[ "$addons_list" != "" ]]; then` here